crates/typos-cli/tests/cmd/force-exclude.toml

```diff
@@ -0,0 +1,6 @@
+bin.name = "typos"
+args = "file.ignore --force-exclude"
```

Comment:
Can you add this test, without the flag, in its own commit, ensure it passes, then implement this feature and update the test?

This helps test the test and show to reviewers that it all works as expected.

Reply:
Sure, I pushed a commit which removes the `--force-exclude` from the tests and should fail the CI.
(Although I forgot to revert the implementation as you suggested, but I'm not sure it's helpful as I assume the commits will be squashed before merge anyway).

Reply:
> l as I assume the commits will be squashed before merge anyway


imo squashing is an anti-pattern.  You either lose valuable information, locked away in a proprietary hosting service or you force people into making non-atomic PRs (refactors that don't make sense without other changes, etc).

Please swap this to the two commit approach *with tests passing in each commit*

Reply:
> Please swap this to the two commit approach _with tests passing in each commit_

Sorry, I misinterpreted your instructions. It's done now.

> imo squashing is an anti-pattern. You either lose valuable information, locked away in a proprietary hosting service or you force people into making non-atomic PRs (refactors that don't make sense without other changes, etc).

To me, squashing serves to remove the commits that are just noise and do not belong to the `main` branch, in particular the ones occurring during the review process (such commits would not exist if the PR was _perfectly_ implemented in the first place).
The alternative is to rewrite the history in response to a comment, but then the whole commit has to be reviewed again instead of just the last change, which is a tedious. The "Compare" feature of Github kind of solves this problem, though.

Anyway, I updated the PR according to your preferences :+1:

Reply:
Thanks!